### DISCUSSION TITLE
Storybook v6.4.7 Global decorators throwing Object are not valid children.

### DISCUSSION THREAD

**Discussion by tejasupmanyu** [storybookjs/storybook#16899] [Help]
Hello,

Recently upgraded to Storybook v6.4.7 from v6.3.8. Since `addDecorator` is now deprecated, I am trying to use a simple decorator from docs like this -

```
// in preview.js

export const decorators = [
  ((Story) => (<div style={{ margin: '3em' }}>
      <Story />
    </div>))
];
```

This is giving me the following error -
<img width="1522" alt="Screenshot 2021-12-04 at 7 21 30 PM" src="https://user-images.githubusercontent.com/17105810/144712188-5072ca98-42d5-436f-bc76-21d7a3036474.png">
 
How to get past this and use the functionality `addDecorator` provided previously? 

Thanks!

**Comment by shilman** [comment]
cc @tmeasday

**Comment by tmeasday** [comment]
Hi @tejasupmanyu that code you posted looks right to me, but I guess I am missing something! Any chance you could post a [reproduction](https://storybook.js.org/docs/react/contribute/how-to-reproduce)?
